micro-benchmarks/DRB201-sync1_yes.c

```diff
@@ -0,0 +1,48 @@
+/*
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+!!! Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+!!! and DataRaceBench project contributors. See the DataRaceBench/COPYRIGHT file for details.
+!!!
+!!! SPDX-License-Identifier: (BSD-3-Clause)
+!!!~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~!!!
+ */
+
+/*
+ * This is a program based on a dataset contributed by
+ * Wenhao Wu and Stephen F. Siegel @Univ. of Delaware.
+
+ * Both threads set and unset the lock before changing the value of x.
```

Comment:
Is your comment accurate?  In the code below, thread with id 0 unsets lock after x=0.